core/src/main/java/nl/nn/adapterframework/stream/Message.java

```diff
@@ -395,6 +397,14 @@ public String asString(String defaultCharset) throws IOException {
 		if (request instanceof String) {
 			return (String)request;
 		}
+		if(request instanceof Node) {
+			try {
```

Comment:
I think it would be wise to not limit the support for Node to the asString() method, and extend it to the other as...() methods.
When you do that, please add tests for that in MessageTest.java

Reply:
Is this used anywhere? If not it might be best to not implement Node at all...